tests/app/questionnaire/test_router.py

```diff
@@ -119,17 +299,20 @@ def test_cant_access_location_not_on_allowable_path(self):
                 "set-duration-units-block",
                 "set-area-units-block",
                 "set-volume-units-block",
-                "summary",
             ],
             section_id="default-section",
         )
+        can_access_location = self.router.can_access_location(
+            current_location, routing_path
+        )
 
-        can_access_location = router.can_access_location(current_location, routing_path)
         self.assertFalse(can_access_location)
 
-    def test_next_location_url(self):
-        schema = load_schema_from_name("test_checkbox")
-        progress_store = ProgressStore(
+
+class TestRouterNextLocation(TestRouterBaseTestCase):
+    def test_section_in_progress(self):
```

Comment:
I was trying to avoid the use of `next_location` etc in the test names when the class is named after a feature. However, I can see it can cause some ambiguities, however, I made some changes to the existing names which should be clearer. Let me know.